internal/lookout/ui/src/components/lookoutV2/JobStatusTable.tsx

```diff
@@ -0,0 +1,55 @@
+import { memo } from "react"
+
+import { Table, TableBody, TableCell, TableContainer, TableHead, TableRow } from "@mui/material"
+import { Job } from "models/lookoutV2Models"
+
+interface JobStatusTableProps {
+  jobsToRender: Job[]
+  totalJobCount: number
+  additionalColumnsToDisplay: {
+    displayName: string
+    formatter: (job: Job) => string
+  }[]
+  showStatus: boolean
+  jobStatus: Record<string, string>
+}
+export const JobStatusTable = memo(
```

Comment:
Just curious, why no tests for this component?

Reply:
@kannon92 It's simple enough that I didn't see there being particularly strong added value in testing it, and it's indirectly tested as part of the two dialog's tests. Would you disagree?

Reply:
Nope that sounds good! Just wanted your thoughts on why